src/components/Editor/components/ProblemHeader/index.js

```diff
@@ -158,6 +140,22 @@ export default class ProblemHeader extends Component {
                         <span id="ProblemMath" className={`${problem.title} ${problem.question}`}>{`$$${this.props.math}$$`}</span>
                         <span id="math-ellipsis" className={`flex-grow-1 ${problem.mathEllipsis}`}>&nbsp;</span>
                         {exampleLabel}
+                        <Button
+                            id="backBtn"
+                            className={classNames('btn', 'pointer', problem.button)}
+                            additionalStyles={['default']}
+                            type="button"
+                            icon="arrow-left"
+                            onClick={this.props.goBack}
+                            tabIndex="-1"
```

Comment:
@rupeshparab Why does the button have a tabindex of "-1"?

Reply:
not sure why the tabindex was added
it was added in this commit along with screen reader text
https://github.com/benetech/MathShare/commit/a6cad4d0b125285f95d01ff14be15c449d5c1a10